tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -87,6 +91,12 @@ Here is a complete class example based on these guidelines:
     func _on_state_changed(previous, new):
         print("state changed")
         emit_signal("state_changed")
+        
+    class State:
```

Comment:
Whatever the order, this should have two lines before it, otherwise it sticks to the method above for no reason.